.github/workflows/e2e.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        k8s-version: [v1.16.15, v1.17.17, v1.18.19, v1.19.11, v1.20.7, v1.21.2, v1.22.5, v1.23.3]
+        k8s-version: [v1.20.15, v1.21.11, v1.22.8, v1.23.5]
```

Comment:
seems like [1.22.7 ](https://hub.docker.com/r/kindest/node/tags?page=1&name=1.22) and [1.21.10](https://hub.docker.com/r/kindest/node/tags?page=1&name=1.21) version are latest available

Reply:
Corrected, thanks!